packages/web-components/src/components/ic-accordion-group/ic-accordion-group.tsx

```diff
@@ -0,0 +1,113 @@
+import {
+  Component,
+  Element,
+  h,
+  Host,
+  Listen,
+  Prop,
+  State,
+} from "@stencil/core";
+import { IcThemeForeground, IcSizes } from "../../utils/types";
+
+let accordionGroupIds = 0;
+@Component({
+  tag: "ic-accordion-group",
+  styleUrl: "ic-accordion-group.css",
+  shadow: true,
+})
+export class AccordionGroup {
+  private accordionGroupId = `ic-accordion-group-${accordionGroupIds++}`;
+
+  @Element() el: HTMLIcAccordionGroupElement;
+
+  @State() accordions: HTMLIcAccordionElement[];
+
+  /**
+   * The header for the accordion group.
+   */
+  @Prop() groupTitle: string = "";
+  /**
+   * The appearance of the tab context, e.g dark, or light.
+   */
+  @Prop() appearance: IcThemeForeground = "default";
+  /**
+   * The size of the accordion.
+   */
+  @Prop() size?: IcSizes = "default";
+  /**
+   * If `true`, the side navigation will load in an expanded state.
+   */
+  @Prop({ mutable: true }) expanded: boolean = false;
+  /**
+   * If `true`,  only one accordion will open at a time.
```

Comment:
Only small, but looks like there's more than one space between the comma and only